packages/block-editor/src/hooks/color.js

```diff
@@ -33,8 +33,19 @@ import useEditorFeature from '../components/use-editor-feature';
 
 export const COLOR_SUPPORT_KEY = '__experimentalColor';
 
-const hasColorSupport = ( blockType ) =>
-	Platform.OS === 'web' && hasBlockSupport( blockType, COLOR_SUPPORT_KEY );
+const hasColorSupport = ( blockType ) => {
+	if ( Platform.OS !== 'web' ) {
+		return false;
+	}
+	const colorSupport = getBlockSupport( blockType, COLOR_SUPPORT_KEY );
+	return (
+		colorSupport &&
+		( colorSupport.link === true ||
+			colorSupport.gradient === true ||
+			colorSupport.background !== false ||
+			colorSupport.text !== false )
+	);
```

Comment:
The same checks  exist on the backend too. `color.php`

Reply:
The file was updated 👍